forestserviceprototype/specialuseform/templates/specialuseform/home.html

```diff
@@ -0,0 +1,14 @@
+{% extends "specialuseform/base.html" %}
+
+{% block content %}
+<h1>Welcome to the Forest Service Online Permits System</h1>
```

Comment:
I think we might not want to pick a system title right now. (I am secretly hoping it doesn't get called "ePermits" 😄 ) What about titling the page **Apply for a permit** ?